sources/packages/web/src/composables/useFormatStatuses.ts

```diff
@@ -0,0 +1,20 @@
+import { ApprovalStatus, GeneralStatusForBadge } from "@/types";
+
+export function useFormatStatuses() {
+  const getProgramStatusToGeneralStatus = (
```

Comment:
As we mentioned in the last call we should get rid of the _GeneralStatusForBadge_.
Just check if makes sense change this code now or when we get rid of it.